libPhoneNumber/NBMetadataHelper.m

```diff
@@ -7,16 +7,8 @@
 //
 
 #import "NBMetadataHelper.h"
-#import "NBGeneratedPhoneNumberMetaData.h"
 #import "NBPhoneMetaData.h"
 
-#import <CocoaLumberjack/CocoaLumberjack.h>
```

Comment:
Is this no longer necessary?

Reply:
The only place where it logged was in decompressing JSON data and finding a non-JSON type. With NSSecureCoding on the archive that's now a hard error (which is okay because the archive is static content and we want to know if it's corrupted).

I guess I can take out the podspec dependency too.